kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -376,7 +372,16 @@ netIPv6ToolTipSubnetMask=For static configuration, enter the subnet mask that wi
 netIPv6ToolTipGateway=For static configuration, enter the IPv6 address of the default gateway for the device.
 netIPv6ToolTipDns=List of DNS servers.<br><br>Servers MUST be separated by a space or a newline.<br><br>When configured for DHCP, manually entered DNS entries will replace the ones provided by the DHCP server. Removing them will restore the DHCP entries.
 netIPv6ToolTipPrivacy=Configure IPv6 Privacy Extensions for SLAAC, described in RFC4941. If enabled, it makes the kernel generate a temporary IPv6 address in addition to the public one generated from MAC address via modified EUI-64. Note that this setting is distinct from the Stable Privacy addresses that can be enabled with the Address Generation Mode property as another way of avoiding host tracking with IPv6 addresses
-netIPv6ToolTipMtu=Selects the Maximum Transition Unit (MTU) for IPv6 traffic over this interface.<br/> A value of 0 allows MTU auto-discovery.<br/>Requires NetworkManager 1.40 or newer.
+
+netAdvanced=Advanced
```

Comment:
Please update also the Japanese translation: https://github.com/eclipse/kura/blob/develop/kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages_ja.properties

Just copy there the Google translate translation, I will make it review by one of our collegues later.

Reply:
Just pushed the japanese labels, will add the docs pr in the next few days.

> the "Network Configuration" section specifying that the advanced tab is available only on generic profiles

Is it specifically generic profiles or only on profiles using Kura's new network management?

Reply:
I would stick with just "generic profiles", as here: http://eclipse.github.io/kura/docs-release-5.4/getting-started/install-kura/